hooks/queries/useMapRoute.tsx

```diff
@@ -44,11 +43,13 @@ export const useMapRoute = ({
     return points;
   };
 
-  const routePoints = getPlanPoints(
-    venuesPlan,
-    { lat: startPoint.center[0], long: startPoint.center[1] },
-    { lat: endPoint.center[0], long: endPoint.center[1] }
+  return useQuery(
+    ["points", venues, startPoint, endPoint, routePoints],
+    () => {
+      return axios.get(
+        `https://api.mapbox.com/directions/v5/mapbox/walking/${routePoints}?geometries=geojson&access_token=${mapboxToken}`
```

Comment:
This architecture is a little strange, might be my doing tbh. Would be good to get the request into its own function then have the hook look something like this:
```
export function useMap() {
  const queryClient = useQueryClient();
  return useQuery({
queryKey: ["points", venues, startPoint, endPoint, routePoints] as const,
    queryFn: () => getRequest,
      });
    },
  });
}
```